apps/cinema-info-server/CinemaServer/Controllers/MainController.cs

```diff
@@ -1,19 +1,39 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using CinemaServer.Data;
+using CinemaServer.Data.Interface;
+using CinemaServer.Entities;
+using CinemaServer.Services;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
 
 namespace CinemaServer.Controllers
 {
     [ApiController]
     public class MainController : Controller
     {
-        [HttpGet("/")]
-        public IActionResult Sesions()
+        CinemaService CinemaService = new();
+        private readonly AppDbContext _context;
+        public MainController(AppDbContext appDbContext)
         {
-            return Json("{Film:123,Dic:210},{Film:123,Dic:210},{Film:123,Dic:210}");
+            _context = appDbContext;
+        }
+        [HttpGet("/main/movies")]       
+        public IActionResult Sesions()
+        {            
+            return Json(CinemaService.MainCinema(_context));
         }
-        [HttpGet("/AllFilms")]
+        [HttpGet("/Add")]
         public IActionResult AllFilms()
```

Comment:
rename and add param -> CreateMovie(MovieRequest movie)
and move body to CinemaService (delegate executing)